mis_builder_budget/models/mis_report_instance.py

```diff
@@ -63,6 +64,7 @@ def _add_column(self, aep, kpi_matrix, period, label, description):
     @api.multi
     def drilldown(self, arg):
         self.ensure_one()
+        arg = safe_eval(arg)
```

Comment:
I think this should not be here. Removed in https://github.com/Eficent/mis-builder/pull/1